Tests/Controller/HelperControllerTest.php

```diff
@@ -460,7 +461,17 @@ public function testSetObjectFieldValueActionWithViolations()
             new ConstraintViolation('error2', null, array(), null, 'enabled', null),
         ));
 
-        $validator = $this->getMock('Symfony\Component\Validator\ValidatorInterface');
+        if (version_compare(Kernel::VERSION, '2.5', '<')) {
```

Comment:
This is not correct, as the kernel version could ne different, if you depend on another Version of this conponent...